From b02692d4fdc858fd011f19b9444f27ea64627f75 Mon Sep 17 00:00:00 2001
From: Florin Sarbu <florin@balena.io>
Date: Fri, 14 Dec 2018 10:33:08 +0100
Subject: [PATCH] Rework resin-specific-env-integration-kconfig.patch

This patch is a rework of the meta-resin patch with the same function because
the meta-resin patch fails to apply due to different source context in
this u-boot version.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 include/env_default.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index 86b639d..b18244f 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -9,6 +9,8 @@
 
 #include <env_callback.h>
 
+#include <env_resin.h>
+
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
 env_t environment __UBOOT_ENV_SECTION__(environment) = {
 	ENV_CRC,	/* CRC Sum */
@@ -21,6 +23,7 @@ static char default_environment[] = {
 #else
 const uchar default_environment[] = {
 #endif
+    BALENA_ENV
 #ifndef CONFIG_USE_DEFAULT_ENV_FILE
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
-- 
2.7.4

